pkg/api/http/metadata.go

```diff
@@ -175,12 +177,13 @@ type metadataActorRuntime struct {
 	Placement    string                           `json:"placement,omitempty"`
 }
 
-type metadataResponsePubsubSubscription struct {
+type MetadataResponsePubsubSubscription struct {
 	PubsubName      string                                   `json:"pubsubname"`
 	Topic           string                                   `json:"topic"`
 	Metadata        map[string]string                        `json:"metadata,omitempty"`
 	Rules           []metadataResponsePubsubSubscriptionRule `json:"rules,omitempty"`
 	DeadLetterTopic string                                   `json:"deadLetterTopic"`
+	Type            runtimev1pb.PubsubSubscriptionType       `json:"type"`
```

Comment:
this I'm not sure if use a string instead of the proto integer value

Reply:
Please, use string. Otherwise, users of the HTTP API will need to consult a documentation to understand each number. For JSON, it is more intuitive to send the text instead of integer for enums.